app/src/main/java/me/capcom/smsgateway/modules/localserver/routes/DocsRoutes.kt

```
@@ -0,0 +1,59 @@
+                call.parameters.getAll("path")?.joinToString("/")
+                    ?.takeIf { it.isNotBlank() } ?: "index.html"
+            val assetPath = "api/$path"
+
+            try {
+                val inputStream = applicationContext.assets.open(assetPath)
+                val bytes = inputStream.readBytes()
+                call.respondBytes(
+                    bytes,
+                    ContentType.defaultForFilePath(assetPath)
+                )
+            } catch (e: IOException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)